centaur/src/main/resources/reference.conf

```diff
@@ -77,5 +77,17 @@ centaur {
     #project = "google-project"
   }
 
+# You need to provide centaur_aws_credentials.conf file with the following structure
+#{
+#    access-key = ""
+#    secret-key = ""
+#    region ="us-east-2"
+#}
```

Comment:
It will work, but instead of keep your aws keys in config file i decided to hide it in centaur_aws_credentials.conf. So you need to provide this structure in this file or in centaur_aws_credentials.conf. Also if you just uncommented this code it would not be any syntax errors, but if you will not provide access and secret keys  and correct the region if needed, centaur will not be able to get in s3 bucket.